src/notation/view/notationviewinputcontroller.cpp

```diff
@@ -171,6 +171,36 @@ void NotationViewInputController::initZoom()
     currentNotation()->viewState()->setMatrixInited(true);
 }
 
+void NotationViewInputController::initCanvasPos()
+{
+    if (currentNotation()->viewState()->zoomType().val != ZoomType::Percentage) {
```

Comment:
When one of the "fit page" zoom options is selected should probably do some centering too:
- When "fit whole page" is selected, the page height is usually the limiting factor, so the page should still be centered horizontally
- In rare cases, when "fit page with" is selected and you open a score with an odd page size, the width will be the limiting factor so the score will still need to be centered vertically

So I would propose to remove this `if`, and adjust `doZoomToPageWidth()`, `doZoomToWholePage()` and `doZoomToTwoPages()` so that they take the `MScore::horizontalPageGapOdd` that is added to all sides into account.

Reply:
Done!